### ISSUE TITLE
docs-bug([FocusTrap]): [Leak documentation about cdkTrapFocusAutoCapture]

### ISSUE THREAD

**Issue by lukaonik** [angular/components#19475]
#### Documentation Feedback

there is a directive called cdkFocusInitial, which is used to focus an initial element, with FocusTrap directive, but the missing point is that if you don't add cdkTrapFocusAutoCapture=true to the FocusTraped element, cdkFocusInitial directive will not work, currently, there is no documentation about that (see  https://material.angular.io/cdk/a11y/overview#regions )
so it would be nice if we could add this kind of info.
we could add it before the example starts: (Remember to use cdkTrapFocusAutoCapture on cdkTrapFocus, for cdkFocusInitial to work)

**Comment by lukaonik** [comment]
I could make the PR with the following changes:
adding note after the example  https://material.angular.io/cdk/a11y/overview#regions 
( cdkFocusTrap needs to have  cdkTrapFocusAutoCapture property to TRUE, in order to focus cdkFocusInitial element automatically upon initialization )

```
<div class="my-inner-dialog-content" cdkTrapFocus  [cdkTrapFocusAutoCapture]="true">
     <input name="email"  cdkFocusInitial />
     <input name="password />
</div>

```
what do you think about that @crisbeto ?
